### DISCUSSION TITLE
Upcoming revamp of our Teacher Swag portal

### DISCUSSION THREAD

**Discussion by samus-aran** [community/community#48175] [GitHub Education]
Hi GitHub Education Educators! 👋

We are excited to announce that this spring we are revamping our Teacher Swag portal!

The GitHub Education team has been working tirelessly behind the scenes on crafting an even better request experience and allowing us to integrate with our existing GitHub Shop. A one-stop swag shop!

As a result, the access to the Teacher Swag request link will be temporarily turned off starting February 27th, 2023. We have further details on the upcoming downtime of the current Teacher Swag portal available on the [Global Campus Teacher’s boards](https://github.com/community/Global-Campus-Teachers/discussions/192) where we will frequently update on the team’s progress and any further developments of the Teacher Swag Program.

We understand that there have been issues with requesting Teacher Swag recently. We would like to direct those still having issues or seeking an update on their status to the latest update posted [here](https://github.com/orgs/community/discussions/46694#discussioncomment-4999443).

Here at GitHub Education we thank you for your continued support and participation in the Teachers Swag program. As a GitHub verified educator please check out the [Global Campus Teacher’s discussion board](https://github.com/community/Global-Campus-Teachers/discussions) to share your feedback and connect with other educators. We greatly appreciate your feedback as it helps us continuously improve and provide the best possible experience for our learners and educators.

**Comment by WJC-POG** [comment]
I've applied almost 20 times but keep getting rejections. I've tried uploading my work documents, course schedule, and employment contract. I am no longer sure what other official documents I need to provide to prove that I am a university teacher.

**Comment by mzunnurainhussain** [comment]
Can you please confirm about the present status of GitHub Swag for teachers?

**Reply by the-rebooted-coder** [reply]
Seems like it’s still paused

**Comment by badmintonstem** [comment]
Hello, is there any update on this yet?

I am hoping to get some stickers to reward my pupils with.

**Comment by kerolosxgad** [comment]
Any updates on this?
@samus-aran

**Comment by jamshaid120** [comment]
any updates on this? @samus-aran 
thanks
